src/main/scala/rps/dto/Response.scala

```diff
@@ -0,0 +1,8 @@
+package rps.dto
+import rps.model._
+
+case class Response(userMove: Move, computerMove: Move, result: Result) {
+  def this(round: Round) {
+    this(round.userMove, round.computerMove, round.result)
```

Comment:
Defining a `this` method on a case class is not very idiomatic. If you want to provide a custom constructor, you usually put it as a "static" method in the companion object, for example:

```scala
case class Response(userMove: Move, computerMove: Move, result: Result)
object Response {
  def apply(round: Round): Response = new Response(round.userMove, round.computerMove, round.result)
}
```

Reply:
if you're really really curious you could try to see how to go from a case class to another with a similar shape using `shapeless`. Warning: not super easy to grok ;)